Tctl disperse info (#1506)
temporalio/documentation: docs/tctl/workflow/describe.md

---

The `tctl workflow describe` command shows information about a [Workflow Execution](/concepts/what-is-a-workflow-execution).

> This information can be used to locate any deployment where a Workflow failed.

Reviewer: What is a "deployment" here?
"... to locate any Workflow Execution that might have failed, for example"

> This information can be used to locate any deployment where a Workflow failed.

Reviewer: ```suggestion
This information can be used to locate a failed Workflow Execution, for example.
```

### `--reset_points_only`

Show only events that are eligible for reset.

> If successful, the command returns the Run Id of all deployments, and the times at which the Event Ids were created.

Reviewer: ```suggestion
If successful, the command returns the Run Id of all deployments, and the times at which the Event Ids were created.
```

Event Ids or .. just Events?
